content/the-ember-times-issue-189.md

```diff
@@ -26,15 +28,15 @@ tags:
 
 ---
 
-## [🐹 2. Section title in sentence case](section-url)
+## [✨ ember-element-query v4](https://github.com/lolmaus/ember-element-query/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Andrey Mikhaylov (@lolmaus)](https://github.com/lolmaus) announced the [release](https://twitter.com/lolmaus_en/status/1424083417591914499) of [ember-element-query](https://github.com/lolmaus/ember-element-query/) v4, a **complete rewrite for Octane** with a new convenient API.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Element queries are a technique to do responsive transformations based on the element's own size, rather than the viewport size. You can implement reusable responsive components with encapsulated styles, that are decoupled from their parent context. Such components will realign their content depending on how much space is available to them.
+
+For example, if you put a responsive component into a tight sidebar, it will align its content vertically. When the sidebar expands, the component will realign horizontally, in order to efficiently use available space. Check out [Element Queries, And How You Can Use Them Today](https://www.smashingmagazine.com/2016/07/how-i-ended-up-with-element-queries-and-how-you-can-use-them-today/) on Smashing Magazine to learn more.
+
+Check out the detailed [README](https://github.com/lolmaus/ember-element-query/) and give the addon a spin! ember-element-query builds on the work started by [ember-fill-up](https://github.com/chadian/ember-fill-up/) and then [ember-container-query](https://github.com/ijlee2/ember-container-query/).
```

Comment:
Not sure if this level of detail is appropriate, but I would like to share some historical context and give due credit.

This suggestion also splits two "Check out..." sentences further apart.

```suggestion
Historically, `ember-element-query` was the first Ember addon to offer element queries out of the box, but back then it used an inefficient implementation and was written in the now-outdated Classic Ember paradigm. It served as one of inspirations for [Chad Carbert's (@chadian)](https://github.com/chadian) [ember-fill-up](https://github.com/chadian/ember-fill-up). Announced on EmberFest 2019 as a proof-of-concept, `ember-fill-up` saw no updates and was discontinued.

The first element query addon to be released with feature completeness, full test coverage and lasting support was [Isaac Lee's (@ijlee2)](https://github.com/ijlee2/) `ember-container-query`, featured in [The Ember Times #153](https://blog.emberjs.com/the-ember-times-issue-153/). The re-release of `ember-element-query` followed suit with a subjectively more convenient, opinionated API, backed by modern, efficient `ResizeObserver` under the hood. [@ijlee2](https://github.com/ijlee2/) provided kind support, criticism and generously shared the demo app with [@lolmaus](https://github.com/lolmaus)'s `ember-element-query`.

Check out the detailed [README](https://github.com/lolmaus/ember-element-query/) and give the addon a spin! 
```

Reply:
On the Conversation tab, the suggested edit has no line breaks. Line breaks are available at the Files changed tab.

Reply:
@lolmaus this is awesome, thank you!